tests/core/shader.test.js

```diff
@@ -0,0 +1,261 @@
+/* global AFRAME */
+/* global assert, process, setup, suite, test, teardown */
+var entityFactory = require('../helpers').entityFactory;
+var componentName = 'shader';
+
+var VIDEO = 'https://ucarecdn.com/bcece0a8-86ce-460e-856b-40dac4875f15/'; // 'base/tests/assets/test.mp4';
+
+function isEmpty (map) {
+  for (var key in map) {
+    return !map.hasOwnProperty(key);
+  }
+  return true;
+}
+
+suite(componentName, function () {
+  setup(function (done) {
+    this.el = entityFactory();
+    done();
+  });
+
+  teardown(function (done) {
+    var el = this.el;
+    process.nextTick(function () {
```

Comment:
can we clean the test file of all the `process.nextTick()`s and `done()` callbacks? 

they're only needed if something asynchronous happens. you also don't need the `done()` callback if nothing async happens.